velox/exec/tests/TaskTest.cpp

```diff
@@ -627,6 +628,34 @@ TEST_F(TaskTest, supportsSingleThreadedExecution) {
   ASSERT_FALSE(task->supportsSingleThreadedExecution());
 }
 
+TEST_F(TaskTest, updateBroadCastOutputBuffersTest) {
+  auto plan = PlanBuilder()
+                  .tableScan(ROW({"c0"}, {BIGINT()}))
+                  .project({"c0 % 10"})
+                  .partitionedOutputBroadcast(std::vector<std::string>{"p0"})
```

Comment:
What is p0 here ? Please can you add a comment what it means ?

Reply:
It is the output layout. It is not required to set it. Leaving it empty.

> @karteekmurthys Looks fine, but the API is confusing.

I have improved it by changing the return type to enum. Please take a look.